doc/book/container/factories.md

```diff
@@ -37,18 +37,18 @@ order to seed the `Application` instance:
 
 - `middleware_pipeline` can be used to seed pre- and/or post-routing middleware:
 
-  ```php
-  'middleware_pipeline' => [
-      // An array of middleware to register prior to registration of the
-      // routing middleware:
-      'pre_routing' => [
-      ],
-      // An array of middleware to register after registration of the
-      // routing middleware:
-      'post_routing' => [
-      ],
+```php
+'middleware_pipeline' => [
+  // An array of middleware to register prior to registration of the
+  // routing middleware:
+  'pre_routing' => [
```

Comment:
No; the original is correct and follows our coding standards (4 space indentation).